### ISSUE TITLE
"The unit 'ppm' is unrecognized" when stitching TASOC light curves

### ISSUE THREAD

**Issue by warrickball** [lightkurve/lightkurve#956]
Sorry, me again...

#### Problem description

I'm getting the error `ValueError: The unit 'ppm' is unrecognized.  It can not be converted to other units.` when trying to stitch a collection of TASOC lightcurves.
```
$ python3 mwe.py
/home/wball/.local/lib/python3.9/site-packages/lightkurve/lightcurve.py:965: LightkurveWarning: The light curve already appears to be in relative units; `normalize()` will convert the light curve into relative units for a second time, which is probably not what you want.
  warnings.warn(
Traceback (most recent call last):
  File "/home/wball/.local/lib/python3.9/site-packages/astropy/units/quantity.py", line 738, in to_value
    scale = self.unit._to(unit)
  File "/home/wball/.local/lib/python3.9/site-packages/astropy/units/core.py", line 950, in _to
    raise UnitConversionError(
astropy.units.core.UnitConversionError: 'UnrecognizedUnit(ppm)' is not a scaled version of 'UnrecognizedUnit(ppm)'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/wball/work/steve/mwe.py", line 5, in <module>
    lc = lcs.stitch()
  File "/home/wball/.local/lib/python3.9/site-packages/lightkurve/collections.py", line 228, in stitch
    return vstack(lcs, join_type="inner", metadata_conflicts="silent")
  File "/home/wball/.local/lib/python3.9/site-packages/astropy/table/operations.py", line 638, in vstack
    out = _vstack(tables, join_type, col_name_map, metadata_conflicts)
  File "/home/wball/.local/lib/python3.9/site-packages/astropy/table/operations.py", line 1299, in _vstack
    col[idx0:idx1] = array[name]
  File "/home/wball/.local/lib/python3.9/site-packages/astropy/units/quantity.py", line 1061, in __setitem__
    self.view(np.ndarray).__setitem__(i, self._to_own_unit(value))
  File "/home/wball/.local/lib/python3.9/site-packages/astropy/units/quantity.py", line 1339, in _to_own_unit
    _value = value.to_value(self.unit)
  File "/home/wball/.local/lib/python3.9/site-packages/astropy/units/quantity.py", line 741, in to_value
    value = self._to_value(unit, equivalencies)
  File "/home/wball/.local/lib/python3.9/site-packages/astropy/units/quantity.py", line 660, in _to_value
    return self.unit.to(unit, self.view(np.ndarray),
  File "/home/wball/.local/lib/python3.9/site-packages/astropy/units/core.py", line 987, in to
    return self._get_converter(other, equivalencies=equivalencies)(value)
  File "/home/wball/.local/lib/python3.9/site-packages/astropy/units/core.py", line 1782, in _get_converter
    raise ValueError(
ValueError: The unit 'ppm' is unrecognized.  It can not be converted to other units.
```

#### Example

```python
import lightkurve as lk

s = lk.search_lightcurve('HV 2112', mission='TESS', author='TASOC', exptime=1800)
lcs = s.download_all()
lc = lcs.stitch()
```
In case it's relevant, I get the same result if I call `lc = lcs.stitch(corrector_func=None)`.

I'm a bit surprised that `ppm` wouldn't be recognised because a quick search turned up [this list](https://docs.astropy.org/en/stable/units/#id25) of AstroPy units, which includes `ppm`.

#### Expected behavior
I'd get a stitched lightcurve (which has so far worked fine with other data HLSPs).

#### Environment

- Fedora 33
- Python 3.9.1
- lightkurve 2.0.0 & astropy 4.2, both via pip

**Comment by warrickball** [comment]
It's certainly a different issue but on the subject of HV 2112, the MAST Portal shows QLP lightcurves for Sectors 1 and 2 but the lightkurve query says both are Sector 1.
```python
>>> s = lk.search_lightcurve('HV 2112', mission='TESS', author='QLP', exptime=1800)
>>> print(s)
SearchResult containing 2 data products.
 #     mission     year author exptime target_name distance
                                  s                 arcsec
--- -------------- ---- ------ ------- ----------- --------
  0 TESS Sector 01 2018    QLP    1800   183306762      0.0
  1 TESS Sector 01 2018    QLP    1800   183306762      0.0
>>> lcs = s.download_all()
>>> print(lcs[0].time[0])
1325.3258788309079
>>> print(lcs[1].time[0])
1354.1383054378064
```
The `time` data definitely implies that the second lightcurve is in Sector 2.

I can open a separate issue for this if you'd like to track it.

**Comment by scfleming** [comment]
Hi Warrick, the sequence number (Sector) is mislabeled for all the QLP Sector 2 entries in Portal as being from Sector 1 by mistake.  We are actively working on fixing that now, should be cleared up soon.  The timestamps are correct, and the sequence number will be fixed imminently.

**Comment by barentsen** [comment]
@scfleming You are a hero for pointing this out. Thank you! ❤️

**Comment by barentsen** [comment]
Regarding the original issue: I started working on a fix over in #959.

The issue is related to the fact that AstroPy's `Table.read()` does not appear to recognize "ppm" as a valid unit.

**Comment by warrickball** [comment]
Is this an issue that should be pushed into AstroPy? I would've thought "ppt" and "ppm" would be popular units for time series outside of Kepler/K2/TESS, too.

**Comment by barentsen** [comment]
Yes, it would be worth discussing on an AstroPy issue!
* `astropy.units.cds.ppm` exists but is not an "enabled" unit by default.
* ppt does not exist at all.

If you are interested, feel free to make the case on an astropy issue!

(Now, regardless of whether or not we can add these to astropy, we'll have to work around it in lightkurve for the time being.)

**Comment by barentsen** [comment]
@warrickball This bug is now fixed via #959 (to be merged once the tests pass). I expect to include this fix in the v2.0.2 release later today.

Demo:

<img width="844" alt="Screen Shot 2021-02-19 at 2 18 24 PM" src="https://user-images.githubusercontent.com/817669/108567593-61cad400-72bd-11eb-861f-b4375f55374c.png">

**Comment by barentsen** [comment]
Lightkurve v2.0.2 has now been released on pypi.  It includes the bugfix for this issue.
